src/helpers/semester.ts

```diff
@@ -0,0 +1,94 @@
+const isSpringSemester = (date: Date) => {
+  const month = date.getMonth();
+  return month > 1 && month < 8;
+}
+
+export class SemesterInfo {
```

Comment:
Please call the file "semester-info.ts" or the like. Kinda confusing with "Semester.vue" also being a thing ^^'